src/RemoteAPI/class-validate-api.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Class for Validate API (`/validate`)
+ *
+ * @package Parsely
+ * @since   3.11.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\RemoteAPI;
+
+use Parsely\Parsely;
+use WP_Error;
+use function Parsely\Utils\convert_to_associative_array;
+
+/**
+ * Class for credentials validation API (`/validate`).
+ *
+ * @since 3.11.0
+ *
+ * @phpstan-import-type WP_HTTP_Request_Args from Parsely
+ */
+class Validate_API extends Remote_API_Base {
+	protected const ENDPOINT     = '/validate/secret';
+	protected const QUERY_FILTER = 'wp_parsely_related_endpoint_args';
```

Comment:
```suggestion
	protected const QUERY_FILTER = 'wp_parsely_validate_secret_endpoint_args';
```
This should be based on the name of the endpoint.